.meteor/versions

```diff
@@ -12,10 +12,6 @@ blaze@2.1.3
 blaze-html-templates@1.0.1
 blaze-tools@1.0.4
 boilerplate-generator@1.0.4
-browser-policy@1.0.5
-browser-policy-common@1.0.4
-browser-policy-content@1.0.6
-browser-policy-framing@1.0.6
```

Comment:
BrowserPolicy lets you set security-related policies that will be enforced by newer browsers. It's not necessarily important in our case that we don't have the package, but if problems were experienced when trying to put the instance inside of a frame, the documentation says to add 

```
BrowserPolicy.framing.allowAll();
```